compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -643,6 +643,7 @@ void addCPUDataTilingPipeline(OpPassManager &passManager,
   {
     GenericVectorizationPassOptions options;
     options.vectorizePadding = true;
+    options.enableVectorMasking = true;
```

Comment:
This change seems like it should be separate.

Reply:
Yes, the change should be landed with https://github.com/openxla/iree/pull/14632

This is just for benchmarking, I will turn it ready for review after landing upstream fix.